src/bump.ts

```diff
@@ -601,13 +606,16 @@ function getNextSdkVer(
     } else {
       // Bumps for "rc" and "dev" are identical on a release branch
       if (currentIsRc) {
-        // Current version is an rc, so bump that
-        nextVersion = currentVersion.nextPrerelease();
-        if (!nextVersion) {
-          die(
-            `Unable to bump RC version for: ${currentVersion.toString()}; ` +
-              `make sure it contains an index number.`
-          );
+        // Only bump if we need to; we don't want to generate a new RC when
```

Comment:
Wouldn't this also apply for releases created on `main`, followed by a fix on the `release/` branch?

Reply:
Yep, that should be handled similarly. Added!